flowtools/peacoqc/PeacoQC_implementation.R

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env Rscript
+
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager")
+
+BiocManager::install("PeacoQC")
+
+library(PeacoQC)
+
```

Comment:
We are probably missing here some optparse or argparse logic here to handle the command line inputs.